### DISCUSSION TITLE
Is there a way to remove the automatic quotations and italics?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#803] [Q&A] [answered]
For example, if we write the following in markdown:
```md
> Example quote
```

Rather than rendering

> Example quote

We get the following

> *"Example quote"*

![image](https://github.com/user-attachments/assets/ecee6ed6-43e4-45f2-bc7b-dd128c723ba5)

If not, I think it'll be more flexibile to not add quotes + italics, and let users do

```md
> *"Example quote"*
```

manually if they want to.

**Comment by fuma-nama** [comment, marked as answer]
This is the default styles of Tailwind CSS Typography, you can customise it with CSS, or replace it with your own component (use `not-prose` utility class to disable typography styles)

**Reply by nktnet1** [reply]
Thank you!
